src/components/mediaLibraryEditor/mediaLibraryEditor.js

```diff
@@ -31,6 +31,14 @@ function onEditLibrary() {
     loading.show();
     const dlg = dom.parentWithClass(this, 'dlg-libraryeditor');
     let libraryOptions = libraryoptionseditor.getLibraryOptions(dlg.querySelector('.libraryOptions'));
```

Comment:
Move this line (`libraryOptions`) after the new block, because otherwise this variable will be set but not used.